src/kw_env.sh

```diff
@@ -135,15 +159,17 @@ function list_env_available_envs()
   local local_kw_configs="${PWD}/.kw"
   local current_env
   local output
+  local ret
 
   if [[ ! -d "$local_kw_configs" ]]; then
     complain 'It looks like that you did not setup kw in this repository.'
     complain 'For the first setup, take a look at: kw init --help'
     exit 22 # EINVAL
   fi
 
-  output=$(find "$local_kw_configs" -maxdepth 1 -mindepth 1 -type d -printf '%f\n' | sort -d)
-  if [[ -z "$output" ]]; then
+  output=$(ls "${local_kw_configs}/${ENV_DIR}" 2> /dev/null)
```

Comment:
Just to keep the consistency around the code, could you replace this `ls` with `find`?